include/gz/sim/components/MultiRay.hh

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2024 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef GZ_SIM_COMPONENTS_MULTIRAY_HH_
+#define GZ_SIM_COMPONENTS_MULTIRAY_HH_
+
+#include <gz/math/Vector3.hh>
+#include <gz/sim/components/Factory.hh>
+#include <gz/sim/components/Component.hh>
+#include <gz/sim/config.hh>
+
+#include <vector>
+
+namespace gz
+{
+namespace sim
+{
+// Inline bracket to help doxygen filtering.
+inline namespace GZ_SIM_VERSION_NAMESPACE {
+namespace components
+{
+/// \brief A struct that holds the information of a ray.
+struct RayInfo
+{
+  /// \brief Starting point of the ray in world coordinates
```

Comment:
If this component is attached to an entity, it's natural to assume the start and end points would be expressed relative to that entity. If the user wants to use world coordinates, they would then just attach the component to the world entity.

Reply:
Agreed.

Done in https://github.com/gazebosim/gz-sim/pull/2514/commits/d4fe7807e20fb974331348fdb6da4d26521c1cbc